controllers/admin/AdminImagesController.php

```diff
@@ -558,6 +603,7 @@ protected function _regenerateNewImages($dir, $type, $productsImages = false)
         }
 
         $generate_hight_dpi_images = (bool) Configuration::get('PS_HIGHT_DPI');
```

Comment:
Maybe you can rename it here at least?
```suggestion
        $generate_high_dpi_images = (bool) Configuration::get('PS_HIGHT_DPI');
```